src/Http/ServerRequest.php

```diff
@@ -589,7 +589,7 @@ public function referer(bool $local = false): string
             }
         }
 
-        return '/';
+        return null;
```

Comment:
What do browsers do if you set `Location: ` header to an empty value?

Reply:
I made a simple test using `<?php header('Location:');` and FF seem to redirect to same page while chrome does nothing.

I think it's upto developer to check the return value if he/she want's to use it for redirection. We have `Controller::redirection()` which returns a default value if no referrer is found for such cases.

Reply:
Good point about Controller::redirect() being able to handle higher level use cases.